Id/Monad

```diff
@@ -0,0 +1,17 @@
+    let Monad = ../Monad/Type 
+
+in  let Id = ./Type 
+
+in  (   { applicative =
+            ./Applicative 
+        , monad =
+            { bind =
+                  λ(a : Type)
+                → λ(b : Type)
+                → λ(fa : Id a)
+                → λ(k : a → Id b)
```

Comment:
This is fine, and probably clearer given the file it’s in, but you could do
```dhall
→ λ(fa : a)
→ λ(k : a → b)
```
instead. `Id` only _needs_ to be used when it’s unapplied to a type.

Reply:
Ya I prefer it this way for readability